test/metabase/upload_test.clj

```diff
@@ -34,30 +33,51 @@
 (def ^:private ai-int-pk-type ::upload/auto-incrementing-int-pk)
 (def ^:private string-pk-type ::upload/string-pk)
 
-(defn- do-with-mysql-local-infile-activated
-  "Helper for [[with-mysql-local-infile-activated]]"
+(defn- local-infile-on? []
```

Comment:
I made these changes because I realised most of the end-to-end CSV tests were run with `with-mysql-local-infile-activated`, which sets `local_infile` to "ON" and runs the test. But we also need to test with `local_infile` "OFF". 

So I replaced the `with-mysql-local-infile-activated` macro with `with-mysql-local-infile-on-and-off`, which tests both.